docs/README.md

```diff
@@ -12,10 +12,10 @@ associated tools first:
 $ sudo pip install sphinx sphinx-autobuild sphinx_rtd_theme livereload==2.2.0
```

Comment:
Also, seeing this new laptop is so fresh and clean I didn't have pip on here so could you please add something like the following to the bottom of the readme?

## Troubleshooting

If you see the following output in your terminal:
```bash
sudo: pip: command not found
```
You'll need to install pip with the following command:
```bash
sudo easy_install pip
```